### PR TITLE
Fix #309: force server URL from build

### LINKED ISSUE #309: Add ability to force server URL when building the admin bundle
**Use case:** be able to send absolute links to particular objects (eg. send an email with the history entries of a particular object).

When we ship the kinto-admin as a server plugin, we could restrict the ability to change the server URL, since we are accessing it via `https:///$SERVER_URL/v1/admin/`.

Alternative approach: provide the server url in the link.

### PR DESCRIPTION
**Pull request by leplatrem**
@n1k0 something like that?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ const corePlugins = [
 
 const settings = {
   maxPerPage: 200,
+  serverURL: "https://kinto.dev.mozaws.net/v1/",
```
Comment (Natim) [inline index.js]: Is there a way to default to the current hostname if we are in `/v1/admin/`?

Comment (n1k0) [inline index.js]: Should be possible by parsing `document.location` indeed.

**Comment by n1k0** [comment]
Looks reasonable :)

**Comment by leplatrem** [comment]
The setting now accepts a string, that we will set with the document.location in the kinto-admin python plugin.
Ready to be reviewed/merged

**Pull request merged**